.ansible-lint

```diff
@@ -0,0 +1,29 @@
+exclude_paths:
+    - releasenotes/
+    - ../
+parseable: true
+quiet: false
+rulesdir:
+    - .ansible-lint_rules/
+verbosity: 1
+# Mocking modules is not recommended as it prevents testing of invalid
+# arguments or lack of their presence at runtime. It is preffered to
+# make use of requirements.yml to declare them.
+mock_roles:
+    - ceph-defaults
+    - ceph-facts
```

Comment:
i think that ceph modules won't be used anymore
cc @fmount

Reply:
+1

Reply:
ditto